tests/pytorch/r2.2/common.libsonnet

```diff
@@ -96,51 +96,13 @@ local volumes = import 'templates/volumes.libsonnet';
         sudo apt install -y libopenblas-base
         # for huggingface tests
         sudo apt install -y libsndfile-dev
-        # TODO change back to torch2.1 once pytorch released torch2.1
-        pip install --user --pre --no-deps torch torchvision --extra-index-url https://download.pytorch.org/whl/test/cpu
-        pip install https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.1.0-cp310-cp310-manylinux_2_28_x86_64.whl
-        pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
-        pip3 install pillow
-        pip3 install typing_extensions
-        pip3 install sympy
-        git clone --depth=1 -b release/2.1 https://github.com/pytorch/pytorch.git
-        cd pytorch
-        git clone -b r2.1 https://github.com/pytorch/xla.git
-      |||,
-    },
-    podTemplate+:: {
-      spec+: {
-        initContainerMap+:: {
-          'tpu-version': null,
-        },
-      },
-    },
-  },
-
-  // TODO: Remove after 2.1 release cut
-  XrtTpuVmMixin:: experimental.PyTorchTpuVmMixin {
-    local config = self,
-
-    tpuSettings+: {
-      softwareVersion: 'tpu-ubuntu2204-base',
-      tpuVmPytorchSetup: |||
-        pip3 install -U setuptools
-        # `unattended-upgr` blocks us from installing apt dependencies
-        sudo systemctl stop unattended-upgrades
-        sudo apt-get -y update
-        sudo apt install -y libopenblas-base
-        # for huggingface tests
-        sudo apt install -y libsndfile-dev
-        # TODO change back to torch2.1 once pytorch released torch2.1
-        pip install --user --pre --no-deps torch torchvision --extra-index-url https://download.pytorch.org/whl/test/cpu
+        pip3 install --user --pre torch torchvision --index-url https://download.pytorch.org/whl/test/cpu
```

Comment:
let's use `pip3 install torch==2.2.0 torchvision torchaudio --index-url https://download.pytorch.org/whl/test/cpu` according to [blog](https://dev-discuss.pytorch.org/t/pytorch-2-2-rc1-is-produced-for-pytorch-audio-vision-and-text/1717)